alpha/veneer/semver/README.md

```diff
@@ -0,0 +1,238 @@
+## Description of the change:
+
+  Since a `veneer` is identified as an input schema which may be processed to generate a valid FBC, we can define a `semver veneer` as a schema which uses channel conventions to facilitate the auto-generation of channels along `semver` delimiters.
+
+### Schema Goals
+The `semver veneer` must have:
+- compact grammar to minimize creation/maintenance effort
+- deterministic output
+- simple channel promotion for maturing bundles
+- demonstration of a common type of channel maturity model
+- Y-stream, X-stream versioning controls
+
+The resulting FBC must clearly indicate how generated channels relate to veneer entities
+
+### Schema Anatomy
+For convenience and simplicity, this veneer currently supports hard-coded channel names `Candidate`, `Fast`, and `Stable`, in order of increasing channel stability.  We leverage this relationship to calculate the default channel for the package. 
+
+`GenerateMajorChannels` and `GenerateMinorChannels` dictate whether this veneer will generate X-stream- or Y-stream-based channels (attributes can be set independently).  If omitted, only minor (Y-stream) channels will be generated.  
+
+Under each channel are a list of bundle image references which contribute to that channel.  
+
+With the (hypothetical) example:
+```yaml
+Schema: olm.semver
+GenerateMajorChannels: true
+GenerateMinorChannels: true
+Candidate:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v0.1.0
+  - Image: quay.io/foo/olm:testoperator.v0.1.1
+  - Image: quay.io/foo/olm:testoperator.v0.1.2
+  - Image: quay.io/foo/olm:testoperator.v0.1.3
+  - Image: quay.io/foo/olm:testoperator.v0.2.0
+  - Image: quay.io/foo/olm:testoperator.v0.2.1
+  - Image: quay.io/foo/olm:testoperator.v0.2.2
+  - Image: quay.io/foo/olm:testoperator.v0.3.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.1
+  - Image: quay.io/foo/olm:testoperator.v1.1.0
+Fast:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v0.2.1
+  - Image: quay.io/foo/olm:testoperator.v0.2.2
+  - Image: quay.io/foo/olm:testoperator.v0.3.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.1
+  - Image: quay.io/foo/olm:testoperator.v1.1.0
+Stable:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v1.0.1
+```
+we define a mock bundle which has 11 versions, represented across each of the channel types.  In this example, `Candidate` has the entire version range of bundles,  `Fast` has a mix of older and more-recent versions, and `Stable` channel only has a single published entry. 
```

Comment:
This is a continuation of the previous clause "With the (hypothetical) example:" not a new thought.  I was trying to avoid using the word 'following' since it's a pet peeve, but I need to keep the full sentence together to help it be clearer.